shared/actions/wallets.js

```diff
@@ -669,14 +669,16 @@ function* walletsSaga(): Saga.SagaGenerator<any, any> {
   )
   yield Saga.safeTakeEveryPure(WalletsGen.accountsReceived, maybeSelectDefaultAccount)
   yield Saga.actionToPromise(
-    [
```

Comment:
maybe easier to read if its just in the handler. thats our typical pattern. we usually listen for N actions but handle cases in the call itself so maybe move this logic to buildPayment and just bail if type is displayCurrencyReceived && !setBuildCurrency in payload

Reply:
Thanks, done